clar2wasm/src/words/blockinfo.rs

```diff
@@ -144,6 +144,7 @@ impl ComplexWord for AtBlock {
 }
 
 #[cfg(test)]
+#[allow(unused_imports)]
```

Comment:
i think it would be better if you instead config-gate the imports as well

Reply:
Removed that directive.